include/deal.II/distributed/p4est_wrappers.h

```diff
@@ -272,7 +272,7 @@ namespace internal
       static
       size_t (&connectivity_memory_used) (types<2>::connectivity *p4est);
 
-      static const unsigned max_level;
+      static const unsigned max_level = P4EST_MAXLEVEL;
```

Comment:
while you're there, can you make it `unsigned int` instead of `unsigned`? Not wrong, just for consistency.

Same in line 457.